csharp/ql/src/Likely Bugs/MishandlingJapaneseEra.ql

```diff
@@ -0,0 +1,60 @@
+/**
+ * @name MishandlingJapaneseEraStartDate
+ * @description Japanese Era should be handled with built-in JapaneseCalendar class. Aviod hard-coding Japanese era start dates and names. 
+ * @id cs/mishandling-japanese-era
+ * @kind problem
+ * @problem.severity warning
+ * @precision medium
+ * @tags reliability
+ *       japanese-era
+ */
+
+import csharp
+
+predicate isExactEraStartDateCreation(ObjectCreation cr) {
```

Comment:
The numbers below are somewhat magical, perhaps a reference to e.g. https://simple.wikipedia.org/wiki/List_of_Japanese_eras in the QL doc? Also, that page seems to suggest that 1st of May 2019 starts a new era, so add that as a case as well?